packages/ibm-products-web-components/src/components/coachmark/coachmark-header.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { LitElement, html, nothing } from 'lit';
+import { property } from 'lit/decorators.js';
+import { prefix } from '../../globals/settings';
+import '@carbon/web-components/es/components/button/index';
+import HostListenerMixin from '@carbon/web-components/es/globals/mixins/host-listener.js';
+import { carbonElement as customElement } from '@carbon/web-components/es/globals/decorators/carbon-element.js';
+import styles from './coachmark-header.scss?lit';
+import Close from '@carbon/icons/es/close/16';
+import Draggable from '@carbon/icons/es/draggable/16';
+import { SignalWatcher } from '@lit-labs/signals';
+import {
+  coachmarkDetailsSignal,
+  updateCoachmarkDetailsSignal,
+} from './coachmark-context';
+import iconLoader from '@carbon/web-components/es/globals/internal/icon-loader';
+
+const blockClass = `${prefix}--coachmark-header`;
+
+/**
+ * coachmark-header for content header section
+ * @element c4p-coachmark-header
+ */
+@customElement(`${prefix}-coachmark-header`)
+class CDSCoachmarkHeader extends SignalWatcher(HostListenerMixin(LitElement)) {
+  /**
+   * Tooltip text and aria label for the Close button icon.
+   */
+  closeIconDescription?: string = 'close icon';
```

Comment:
it's best not to have hard coded strings even for default values. just `''` is sufficient.